src/icons/general/IconStorage.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    title?: string,
+    width?: number,
+};
+
+const IconStorage = ({ className = '', color = '#3FB87F', height = 12, title, width = 16 }: Props) => (
```

Comment:
proper color

Reply:
do you mean to use any of the color from `/styles/variables.js`

Reply:
Ask the designer to review the palette

Reply:
yes checking with her. thanks

Reply:
updated the PR with bdlGreenLight